contrib/cirrus/pr-should-link-jira.t

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
```

Comment:
To ensure the script doesn't break in the future, it would be good to execute the tests as part of CI.  Right after the `lib.sh.t` test would be a great place:

https://github.com/containers/podman/blob/main/contrib/cirrus/prebuild.sh#L62-L64

Reply:
Ah, no it fails because GITHUB_TOKEN is not set

Reply:
Fixed it, should work now.